### ISSUE TITLE
Wrong schematic for IP5306?

### ISSUE THREAD

**Issue by ModischFabrications** [wagiminator/Power-Boards#3]
The reference implementation in http://www.injoinic.com/wwwroot/uploads/files/20200221/0405f23c247a34d3990ae100c8b20a27.pdf shows the resistor between BAT and the battery connector, but https://github.com/wagiminator/Power-Boards/blob/master/LiPo_Power_Board_IP5306/LiPo_Power_Board_IP5306_schematic.pdf has R4 towards ground. There are also a few deviating values. 

I'm not sure if I'm missing something or if this is an actual mistake, feel free to close if the former. Could you give a few details on why you chose these values? I'm not finding accepted value ranges or external part specs.

**Comment by wagiminator** [comment]
Yes you are right. There is a small flaw, the battery feedback after the resistor would be a little less prone to noise. But it also works fine this way.
